drivers/i2c/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ else()
 	)
 endif()
 zephyr_library_sources_ifdef(CONFIG_I2C_NUMAKER		i2c_numaker.c)
+zephyr_library_sources_ifdef(CONFIG_I2C_NXP_II2C	i2c_nxp_ii2c.c)
 zephyr_library_sources_ifdef(CONFIG_I2C_SAM_TWI		i2c_sam_twi.c)
```

Comment:
Is this a typo or the naming is actually II2c and not I2C?

Reply:
It is I2C, but we call this IP on imx8m series with the name ii2c, same as GPIO on imx8m series is igpio, we use the same driver name in hal_nxp.